### DISCUSSION TITLE
Sensor chart

### DISCUSSION THREAD

**Discussion by bern6** [mytourbook/mytourbook#1095] [Q&A]
Just a quick question, what is the difference between vertical line and dot.

Thanks

![Sensor chart](https://user-images.githubusercontent.com/103836286/237024265-8e8ffa00-bc0b-4f53-9f24-12efb9183210.png)

**Comment by wolfgang-ch** [comment]
Your battery values are saved in 10% steps, e.g. 100%, 90%, 80%, ...

When a tour has saved the battery value only for one 10% step, e.g. 70%, then a dot is displayed and when the tour spans multiple 10% values, then a line is displayed, e.g. 70%...60%
